src/image_gen/imagegenutils.cpp

```diff
@@ -127,7 +127,67 @@ std::variant<absl::Status, std::optional<size_t>> getSizetFromPayload(const ovms
     SET_OR_RETURN(std::optional<TYPE>, value, FUNCTION(payload, key)); \
     INSERT_IF_HAS_VALUE_RETURN_IF_FAIL(key, value);
 
+absl::Status ensureAcceptableForStatic(ov::AnyMap& requestOptions, const ovms::ImageGenPipelineArgs& args) {
```

Comment:
There's a lot of conversion calls `as<type>()` 
Are they all guaranteed to be safe?

Reply:
checked and it is fine - we catch the exception